src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/ProductVariantImage.orm.xml

```diff
@@ -18,6 +18,8 @@
 
     <mapped-superclass name="Sylius\Component\Core\Model\ProductVariantImage" table="sylius_product_variant_image">
         <many-to-one field="variant" target-entity="Sylius\Component\Product\Model\VariantInterface" inversed-by="images" />
+        <field name="title" column="title" type="string" nullable="true" />
```

Comment:
IMO fields should be declared before relations, same as it's done in other model declarations.